resources/qml/Toolbar.qml

```diff
@@ -246,7 +246,7 @@ Item
             }
             PropertyChanges {
                 target: panelBorder
-                anchors.bottomMargin: -(base.activeY + UM.Theme.getSize("button").height)
+                anchors.bottomMargin: ((base.activeY + UM.Theme.getSize("button").height) > panelBorder.height) ? -(base.activeY + UM.Theme.getSize("button").height) : -panelBorder.height
```

Comment:
Suggestion for readability:
```
                anchors.bottomMargin: 
                {
                    if (panelBorder.height < base.activeY + UM.Theme.getSize("button").height)
                    {
                        return base.activeY + UM.Theme.getSize("button").height
                    }
                    return panelBorder.height
                }
```

Reply:
I think your suggested code is missing some - signs but I get the idea. Will do.

I didn't realise that in qml you can use return within curly brackets like that.

Reply:
Now beautiful!

Reply:
Gorgeous!